reconcile/blackbox_exporter_endpoint_monitoring.py

```diff
@@ -159,6 +167,24 @@ def fill_desired_state(provider: EndpointMonitoringProvider,
 @defer
 def run(dry_run: bool, thread_pool_size: int, internal: bool,
         use_jump_host: bool, defer=None) -> None:
+    # verify blackbox-exporter modules
+    settings = queries.get_app_interface_settings()
+    allowed_modules = settings["endpointMonitoringBlackboxExporterModules"]
```

Comment:
Given how you use this below, you may want to turn it into a `set`.

Reply:
i'm not using `sets` often enough. thanks for the reminder